File: app/src/main/java/com/BlizzardArmory/warcraft/equipment/Weapon.java
package com.BlizzardArmory.warcraft.equipment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Weapon {

    @SerializedName("damage")
    @Expose
    private Damage damage;
    @SerializedName("attack_speed")
    @Expose
    private AttackSpeed attackSpeed;
    @SerializedName("dps")
    @Expose
    private Dps dps;

    public Damage getDamage() {
        return damage;
    }

    public void setDamage(Damage damage) {
        this.damage = damage;
    }

    public AttackSpeed getAttackSpeed() {
        return attackSpeed;
    }

    public void setAttackSpeed(AttackSpeed attackSpeed) {
        this.attackSpeed = attackSpeed;
    }

    public Dps getDps() {
        return dps;
    }

    public void setDps(Dps dps) {
        this.dps = dps;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/Weapon.kt
package com.BlizzardArmory.model.warcraft.equipment

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Weapon.
 */
@Keep
data class Weapon(

    @SerializedName("damage")
    var damage: Damage,

    @SerializedName("attack_speed")
    var attackSpeed: AttackSpeed,

    @SerializedName("dps")
    var dps: Dps

)